tensorflow/lite/micro/kernels/circular_buffer.cc

```diff
@@ -54,6 +54,11 @@ namespace {
 constexpr int kInputTensor = 0;
 constexpr int kOutputTensor = 0;
 
+// Indices into the init flexbuffer's vector.
+// The parameter's name is in the comment that follows.
+// Elements in the vectors are ordered alphabetically by parameter name.
```

Comment:
can you expand this comment to describe how this ordering is achieved? Does the flexbuffer implementation enforce this or something else?

Reply:
Done in flatbuffer_utils.h